Makefile

```diff
@@ -41,8 +41,8 @@ import: inputs/efile_COAK_2016_A-Contributions.csv inputs/oakland_candidates.csv
 	./latest_only.sh efile_COAK_2016_Summary
 	./remove_duplicate_transactions.sh
 
-inputs/efile_COAK_%_A-Contributions.csv: downloads/efile_COAK_%.xlsx
-	ssconvert -S $< inputs/$(subst .xlsx,_%s.csv,$(shell basename $<))
+inputs/efile_COAK_2016_A-Contributions.csv: downloads/efile_COAK_2016.xlsx
+	ruby ssconvert.rb $<
 
```

Comment:
The % placeholders in the Makefile were confusing me, and I don't think we ever used the fact that this would have been able to download multiple years' data anyway. (Though, now we need to do that since there _is_ data from 2017.)